instrumentation/instasarama/consumer_group_handler_test.go

```diff
@@ -23,6 +23,15 @@ func TestConsumerGroupHandler_ConsumeClaim(t *testing.T) {
 		{
 			Topic: "topic-1",
 			Headers: []*sarama.RecordHeader{
+				{
```

Comment:
I'm entirely not sure about any of the changes in the tests for the *consumers*. Apparently we add headers to messages on the consumer side when no headers have been present in the original incoming message? That concept was new to me and a bit suprising. But from what I understand it is used so that the customers' applications can read the headers and continue the trace from the trace ID/span ID in the headers, is that correct?

If that understanding is correct, the header names and their contents are actually part of our public API in go and we need to make customers aware of this change, so they can update their application code somewhere between October 2022 and October 2023.